template.go

```diff
@@ -79,19 +80,42 @@ func readYAMLFile(yamlTemplate string, templateArgs map[string]string, opts *Opt
 	return readYAMLRequest(base, contents, templateArgs, opts)
 }
 
+func getYAMLRequestBody(t template, templateArgs map[string]string) ([]byte, error) {
+	if t.Request != nil {
+		req, err := templateargs.ProcessMap(t.Request, templateArgs)
+		if err != nil {
+			return nil, err
+		}
+
+		return yaml.Marshal(req)
+	}
+
+	var body string
```

Comment:
nit: you could use a string builder here (faster for string concatenation). This code is not in a hot path, so only nit

Reply:
good one, this didn't require string use actually so changed body to `[]byte` which now uses `append`